### ISSUE TITLE
Reference ontologies vs. "Other" kinds

### ISSUE THREAD

**Issue by matentzn** [OBOFoundry/OBOFoundry.github.io#1546]
There are many kinds of ontologies with many different intentions. Whenever I tried to suggest naming them, it caused too much debate, but I would like to at least distinguish a few fundamentally different "kinds" of semantic artefacts. In the following, the word `domain` means "ontological branch" rather than "topic", and "modelling a domain" means "providing a detailed hierarchy of a `domain` and its relationships to other domains" (I am not trying to redefine these terms, just setting the scene for the following):

- Ontologies that model one or more domains (lets call them domain ontologies just for the sake of argument):
   - Disease Ontology models the domain of diseases
   - Uberon models the domain of anatomical entities
   - GO models the domain of biological processes (among others)
   - OBI models the domain of assays (among others)
   - ENVO models the domain of environmental materials (among others)
- Ontologies that extend another ontology by being more fine-grained
   - XAO extends (conceptually) Uberon
   - ZFA extends Uberon
   - Note that while there is danger in terms of scope overlap between higher and lower level extensions, these are generally save.
   - Extensions should be flagged as such and be integrated in the main "upper" domain ontologies like the ones above.
   - _Extensions should only be permitted in OBO if the do-_main_ ontologies explicitly say "out of scope"_
- Ontologies that collect terms from domain ontologies to serve a specific application (there are many kinds, here some examples):
   - COVOC is a collection of terms from domain ontologies for the purpose of annotating a corpus of COVID literature automatically using NLP
   - DISDRIV is a collection of terms from domain ontologies for the purpose of annotating drivers of disease
   - Epilepsy Ontology is a collection of terms from domain ontologies for the purpose of... (annotating epilepsy papers? just musing)
   - While it's hard to draw clear lines here, three features are clear: 
      1. The ontologies do not intend to model a domain (in my sense of the words, see above), even if they for various reasons happen to add terms
      2. Building logical dependencies on such ontologies is risky for OBO as a whole, as they tend to change the axiomatisation of upstream ontologies by linking them for a specific purpose.
      3. They are _super valuable_ as they are integration points of different ontologies, help us to understand the compatibility of different ontologies and encourage OBO term re-use. 

I hereby want to propose to establish a clear distinction of the three types above with the hope of convincing tools like OLS to flag these ontologies according to this configuration. Concretely, I would like:
1. Category 1 ontology terms (domain ontologies) to be sorted before the others, so the terms in the search index should get precedence.
2. Category 1 and 2 ontologies being rigorously checked for the "overlapping domain violation" during OBO review, and refused on the grounds of violating that.
3. Category 1 and 2 ontologies being examined on the basis of OBO modelling patterns: COB-ness, BFO-ness, etc
4. Category 3 ontologies to be excluded from search of term browsers, and of you go to the ontology directly, being visually warned to not re-use terms.

Also before we start a name war, invent some new terms that define the three types.

**Comment by alanruttenberg** [comment]
First reaction: I'm not sure the distinction between domain and extension ontologies is clear and accurate. Some "extension" ontologies are clearly domain ontologies on their own. Also, lumping extensions with the domain ontology drags down their status. The example given, Uberon vs extensions is not a good one.  Concepturally, Uberon originated as an extension (to more general), being based on the original anatomy ontologies, and then refined. Second, to the extent that we don't like injecting axioms, Uberon does just that - adding subclass relations between Uberon terms and more terms in the more specific anatomies.

So, I would erase that distinction and instead include all domain ontologies in one class, with our usual criteria for domain ontologies. It's not that it's a wrong label, it's that policy that distinguishes the two, like changes in browsing or search, would be, properly, insulting to the developers of those ontologies.

We could add, as one fact about an ontology, that it is finer grained than another but have matching inverse statements that one is more general than the other.

Other thoughts:

All ontologies should be evaluated wrt your (3).

Regarding point 2, are we refusing any ontologies at the moment? I thought we were making better information available about ontologies (dashboard) and trying to influence them (discussion on submission to be listed) to do the right thing, but not refusing them. I'm not following OBO as closely as I used to, so perhaps I missed a policy change.

The risks you list are indeed risks, but it would make more sense to have these documented, perhaps with links beside the category 3 terms that lead to documentation of those risks. 

In my view, category 3 ontologies should not be excluded from term browsers. If they are so bad as to want to exclude them there then exclude them from the library altogether. Or, if the idea of a preferred set of foundry ontologies comes back into currency with strong consensus recommendations that they be used, _then_ the other terms can be de-emphasized (sort order, font color) but not get excluded. In other words. I think we either go whole hog with a limited, sanctioned list of ontologies, or we don't. Which is where I thought we are now.

**Comment by matentzn** [comment]
I am happy with merging the top two categories into one - the distinction came up during the OBO operations committee calls and that's why I included it here.

Maybe I should clarify my point.

1) we want to _encourage category 3 ontologies_ as they demonstrate usage, reuse of OBO ontologies and links the OBO foundry to the users. It promotes a culture of community orientation. People want OBO expert eyes on their work and our nod of approval. We do not want to reject them on the grounds of them being category 3 ontologies.
2) Category 3 ontologies are not suitable sources of terms or axiomatisation. They don't have a proper scope. They inject axioms that are false.  And we don't have the time resources to fix every single axiom, term etc in every submission - and even if we somehow would be able to do so, what then? Say an ontology is _just_ a set of locality-based modules merged together to satisfy some use case. We want people to do that (see point 1). But there is no point building dependencies on such ontologies. 

My impression during the committee call was that we have nearly unanimous agreement that we need some kind of distinction - we just couldn't decide yet how to draw the lines..

**Comment by cmungall** [comment]
@alanruttenberg has some good points regarding distrinctions between 1+2. In the interest of incremental improvement, perhaps let us group 1+2 together. However, I think it's a great idea to make an independent proposal to add a tag to our metadata to link species-specific ontologies (e.g ZFA) to species-general ontologies (uberon), where there exists some bridge files with axioms of the form X = Y and in-taxon some T. Browsers are then free to use that to rank search results as they please.

regarding as to whether 3 should be excluded from search, I think that is a decision for the various browsers. We can give recommendations but that is up to the individual browsers. I would expect OLS will keep showing EFO anyway. But I do think OBO should recommend that search results for 3 should appear before 1+2, and there should be some visual indication of the category. Certainly within the OBO list we would list these after 1+2 (but before obsoletes and inactives).

Regarding naming:

 - I favor "compilation ontology" for 3.
 - for 1+2:
      - "core" makes sense but has been over-used
      - "original"? problematic as being graded non-original would be pejorative
      - "source ontology" - I think this is my preference

**Comment by matentzn** [comment]
Sorry we responded at the exact same second :D

**Comment by alanruttenberg** [comment]
@matentzn I agree with your assessment of level 3 ontologies. What I was attempting to  address is was what to do about that. There was a long discussion, over the years, about whether the foundry should advertise just the foundry-reviewed ontologies or list any ontology that was open. It was decided that the latter would be our policy. Over the years we have gently increased our feedback and attempts to raise standards in gentle ways (thanks!!).  It is with respect to that practice that I was objecting to things like de-listing.

**Comment by alanruttenberg** [comment]
Another thing that could be done would be to add a check to the dashboard on whether an ontology does, in fact, add axioms to such terms. We would then discuss, during submission, the downsides of that practice in a similar way as we would other dashboard dings.

**Comment by alanruttenberg** [comment]
I just looked at the principles and I don't think we have one addressing the issue, so that's another opportunity.

**Comment by nlharris** [comment]
I think this will be addressed by the new home page ontology tables, right?

**Comment by matentzn** [comment]
No but I personally give up on this issue after having derailed 12 ofoc calls insisting on it. I think a different distinction is more easy to argue with OFOC which is: fits under COB, fully consistent with dependencis, no axiom injection and no terms that belong elsewhere. I think this issue served its purpose and can be closed
